index.html

```diff
@@ -0,0 +1,335 @@
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>ERB</li>
+              <li>RSPec</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/katchy/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <section class="contact flex" id="contact">
+        <p class="parag text-dark">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
```

Comment:
- [ ] [OPTIONAL] Nice work adding this header to your form section, however there are some differences between your design and the Figma design, please try to make your design looks like the Figma template as much as possible as stated in [equirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) 🔽
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Figma design 
![dddd](https://user-images.githubusercontent.com/104319462/218207855-9ac06e2e-2b12-4e22-aa23-b3b39132f76f.png)
### Your design 
![cccc](https://user-images.githubusercontent.com/104319462/218207877-92cca69d-1d44-41ad-9773-f832692015c2.png)

- [ ] [OPTIONAL] Also please consider using a more semantic tag like a `<header>` instead of a `<p>` as it follows HTML best practices and makes your project more accessible.